Package.swift

```diff
@@ -27,5 +31,26 @@ let package = Package(
         .testTarget(
             name: "SymbolKitTests",
             dependencies: ["SymbolKit"]),
+        .executableTarget(
+            name: "dump-unified-graph",
+            dependencies: [
+                "SymbolKit",
+                .product(name: "ArgumentParser", package: "swift-argument-parser"),
+            ]),
     ]
 )
+
+// If the `SWIFTCI_USE_LOCAL_DEPS` environment variable is set,
+// we're building in the Swift.org CI system alongside other projects in the Swift toolchain and
+// we can depend on local versions of our dependencies instead of fetching them remotely.
+if ProcessInfo.processInfo.environment["SWIFTCI_USE_LOCAL_DEPS"] == nil {
```

Comment:
Should we just exclude the `dump-unified-graph` target when building in SwiftCI?

Reply:
How do we do that? Does that mean dropping it from the `products` listing? Once that's done, would i rework the dependencies block to always pull ArgumentParser from GitHub?

Reply:
Ah I think we would essentially keep this block but remove the `else` statement and add:

```swift
package.targets += [
    .executableTarget(
         name: "dump-unified-graph",
         dependencies: [
             "SymbolKit",
             .product(name: "ArgumentParser", package: "swift-argument-parser"),
         ]),
]
```

Does that make sense? It's definitely a _little_ odd but I think worth it to reduce risk in SwiftCI.

Reply:
After looking at it a little bit, i thought it would look like that. I've pushed up a couple commits to address your comments. ✌️